plenoptic/simulate/models/naive.py

```diff
@@ -235,27 +237,28 @@ def __init__(
 
         self.cache_filt = cache_filt
         self._filt = None
+        self._input_device = torch.device("cpu")
 
     @property
     def filt(self) -> Tensor:
         """Creates an on center/off surround, or off center/on surround conv filter"""
         if self._filt is not None:  # use cached filt
             return self._filt
         else:  # generate new filt and optionally cache
-            filt_center = circular_gaussian2d(self.kernel_size, self.center_std, self.out_channels)
-            filt_surround = circular_gaussian2d(self.kernel_size, self.surround_std, self.out_channels)
             on_amp = self.amplitude_ratio
+            device = on_amp.device
+
+            filt_center = circular_gaussian2d(self.kernel_size, self.center_std, self.out_channels, device)
+            filt_surround = circular_gaussian2d(self.kernel_size, self.surround_std, self.out_channels, device)
 
             # sign is + or - depending on center is on or off
-            sign = torch.as_tensor([1. if x else -1. for x in self.on_center])
+            sign = torch.as_tensor([1. if x else -1. for x in self.on_center]).to(device)
             sign = sign.view(self.out_channels, 1, 1, 1)
-            filt = on_amp * (sign * (filt_center - filt_surround))
 
-            filt = filt / filt.sum()
+            filt = on_amp * (sign * (filt_center - filt_surround))
```

Comment:
why did we get rid of  the normalization of the filter?

Reply:
It didn't really make sense bc the filter contains both pos and neg values, so it isn't clear what dividing by its sum would achieve.